internal/build/build.go

```diff
@@ -64,12 +64,13 @@ func needLLFile(mode Mode) bool {
 }
 
 type Config struct {
-	BinPath   string
-	AppExt    string   // ".exe" on Windows, empty on Unix
-	OutFile   string   // only valid for ModeBuild when len(pkgs) == 1
-	RunArgs   []string // only valid for ModeRun
-	Mode      Mode
-	GenExpect bool // only valid for ModeCmpTest
+	BinPath      string
+	AppExt       string   // ".exe" on Windows, empty on Unix
+	OutFile      string   // only valid for ModeBuild when len(pkgs) == 1
+	RunArgs      []string // only valid for ModeRun
+	Mode         Mode
+	GenExpect    bool // only valid for ModeCmpTest
+	BuildOptions BuildOptions
```

Comment:
move this field before `GenExpect` for byte alignment.

Reply:
Removed BuildOptions